package oc.a.chapters._6_exceptions.usingATryStatement.throwingASecondException.onlyTheLastExceptionGetsThrown;

class FinallyException extends Exception{

}
